scripts/EN/Harry

```diff
@@ -3513,9 +3513,9 @@ CleanScreen(); // 0x4428: 00000064
 LoadBackground(0x54); // 0x442c: 00001069
 DrawBackground(); // 0x4432: 00000065
 PlayMusic(0x6e, 0x1); // 0x4436: 0000206c
-ShowText((4058)"I...get out the pet bottle and use a fragment of the bottom section of one of the broken bottles to pour the purple liquid into the it."); // 0x443e: 0000100c
+ShowText((4058)"I…get out the plastic bottle and use a fragment of the bottom section of one of the broken vials to pour the purple liquid into the it."); // 0x443e: 0000100c
 HandleInput(); // 0x4444: 00000010
-ShowText((4059)"The sticky liquid drips into the pet bottle. The last drop lingered, but once it finally hit the surface of the liquid already inside the pet bottle I tightly closed the cap and then left the Director's Office."); // 0x4448: 0000100c
+ShowText((4059)"The sticky liquid drips into the plastic bottle.¥nW2The last drop lingered, W1but once it finally hit the surface of the liquid already inside the plastic bottle W1I tightly closed the cap and then left the Director's Office."); // 0x4448: 0000100c
```

Comment:
Changed pet to plastic to match the PS1 version. (PET refers to the type of plastic and is how they refer to these bottles in Japan.)